packages/comps-db/src/schema.ts

```diff
@@ -116,7 +116,7 @@ export const balances = pgTable(
     updatedAt: timestamp("updated_at", {
       withTimezone: true,
     }).defaultNow(),
```

Comment:
You'll have to generate a migration to get this change to show up in the db. If we apply that migration to the existing db, you might have to edit the migration SQL to set some value for rows that have a `null` specific chain. Does that sound reasonable? We should talk about this more, but if we don't want to ever migrate the existing database because we just don't need to or because it's too complicated, then we can just ignore migrations for now and make whatever changes to the schema we want.